API.md

```diff
@@ -152,3 +152,41 @@ The payload should be a classifier object, with a classifier type selector `name
 An `HTTP 404 Not Found` status code is returned if the survey with the specified ID could not be found.
 
 An `HTTP 409 Conflict` status code is returned if a classifier type selector already exists for any of the names in the payload.
+
+## Post New Survey
+* `POST /surveys` will create a new survey.
+
+The payload should be a byte array, with a `ShortName`, a `LongName`, a `SurveyRef`, a `LegalBasis`, and a `SurveyType`.
```

Comment:
From looking through the code shouldn't the payload match this struct?
```
// Survey represents the details of a survey.
type Survey struct {
	ID            string `json:"id"`
	ShortName     string `json:"shortName" validate:"required,no-spaces,max=20"`
	LongName      string `json:"longName" validate:"required,max=100"`
	Reference     string `json:"surveyRef" validate:"required,max=20"`
	LegalBasis    string `json:"legalBasis"`
	SurveyType    string `json:"surveyType"`
	LegalBasisRef string `json:"legalBasisRef"`
}
```

Reply:
The same struct is also used for the request below.

Reply:
The payload should be a `json` doc (the fields of which match those in the struct)